processing/src/main/java/io/druid/query/AsyncQueryRunner.java

```diff
@@ -49,7 +49,7 @@ public AsyncQueryRunner(QueryRunner<T> baseRunner, ExecutorService executor, Que
   @Override
   public Sequence<T> run(final Query<T> query, final Map<String, Object> responseContext)
   {
-    final int priority = BaseQuery.getContextPriority(query, 0);
+    final int priority = QueryContexts.getPriority(query, 0);
```

Comment:
getPriority is always called with 0 as the default argument, maybe remove the parameter